server/controllers/card.js

```diff
@@ -0,0 +1,78 @@
+const User = require("../models/User");
+const Board = require("../models/Board");
+const Column = require("../models/Column");
+const asyncHandler = require("express-async-handler");
+
+
+exports.createCard = asyncHandler(async (req, res, next)=> {
+
+  if (!req.body) {
+    res.status(204);
+    throw new Error("Input missing. please fill up all the fields.");
+  }
+
+  const {columnId, boardId, name, tag} = req.body
```

Comment:
What if there is partial data passed down like no boardId? Ideally we want to do input validation on the back-end with middleware like express-validator.

Reply:
Added validation using express-validator.